pipewrench/merge_test.py

```diff
@@ -16,6 +16,7 @@
 """
```

Comment:
This test should be renamed to reflect that it's only a test now for the map_datatypes plugin/function